src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ReconfigureFundingCycleSettingsPage/hooks/initialEditingData.ts

```diff
@@ -120,7 +121,7 @@ export const useInitialEditingData = ({
   }
 
   // Populates the local redux state from V2V3ProjectContext values
-  useEffect(() => {
+  useDeepCompareEffect(() => {
```

Comment:
This useEffect block was being re-run on every re-render, because it's dependencies are objects. Using useDeepCompareEffect should fix it maybe.

https://github.com/kentcdodds/use-deep-compare-effect